app/javascript/dashboard/components/widgets/conversation/Message.vue

```diff
@@ -66,6 +65,7 @@
       v-if="isBubble"
       :is-open="showContextMenu"
       :show-copy="hasText"
+      :show-delete="showDelete"
```

Comment:
I guess we can disable the entire option. Copy doesn't make sense if the message is already deleted.